src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2554,9 +2562,17 @@ class CgroupUtils(object):
         except KeyboardInterrupt:
             raise
         except:
-            if 'systemd' in self.paths:
-                # Must be at least version 205 if the cgroup is present
-                return 205
+            # Loop through the mounts to see if systemd is present. Do not
+            # rely on self.paths because it has not been initialized yet.
+            with open(os.path.join(os.sep, 'proc', 'mounts'), 'r') as desc:
+                for line in desc:
+                    entries = line.split()
+                    if entries[2] != 'cgroup':
+                        continue
+                    flags = entries[3].split(',')
+                    if 'systemd' in flags or 'name=systemd' in flags:
```

Comment:
I don't believe that is correct because 'systemd' will not match 'name=systemd' and vice-versa. We need to explicitly check for both. On my system the line we start with is...
```cgroup /sys/fs/cgroup/systemd cgroup rw,nosuid,nodev,noexec,relatime,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd 0 0```

Reply:
Oh, it's a list.  I thought it was a string.  If it was a string, in would work.  Since it is a list of strings, you do need the exact string.

Reply:
Yes. Will respond to more comments tomorrow.